src/views/VerifyPrevAddress.vue

```diff
@@ -4,11 +4,12 @@
       <ProgressBar :nodeData="progressBarData"/>
       <h1>Verify you own {{alias}}@nav.community</h1>
       <div class="input-container">
-        <ListEntry><span slot="number" class="number">1</span><span slot="text" class="text small">Copy the message below</span></ListEntry>
+        <ListEntry><span slot="number" class="title number">1</span><span slot="text" class="text small">Copy the message below</span></ListEntry>
         <Copybox>signmessage {{aliasCurrentAddress}} {{alias}}@nav.community</Copybox>
         <br /> <br />
+        <br /> <br />
         <!-- <div><Button @click="copyText()">{{copied ? "Copied" : "Copy"}}</Button></div> -->
-        <ListEntry><span slot="number" class="number">2</span><span slot="text" class="text small">Now open your NavCoin wallet, open console through Help -> Debug and pastes the message, then hit Enter on your keyboard. An response message (verification code) will be created, copy and paste it here</span></ListEntry>
+        <ListEntry><span slot="number" class="title number">2</span><span slot="text" class="text small">Now open your NavCoin wallet, open console through Help -> Debug and pastes the message, then hit Enter on your keyboard. An response message (verification code) will be created, copy and paste it here</span></ListEntry>
```

Comment:
I don't think we should actually have this here, because its a different process depending on the if its NavPi or not.

Maybe we could just say 'Paste the response here. See below for detailed instructions' ?

Otherwise, just correct the grammer/spelling.

```
Open your NavCoin wallet, open console through the Help -> Debug menus. 
Pastes the above message, then press 'enter' on your keyboard. 
The response message (verification code) will be created. 
Copy and paste the message below.
```

Reply:
fixed